docs/library/network.WLAN.rst

```diff
@@ -133,4 +133,24 @@ Methods
    hostname       The hostname that will be sent to DHCP (STA interfaces) and mDNS (if supported, both STA and AP). (Deprecated, use :func:`network.hostname` instead)
    reconnects     Number of reconnect attempts to make (integer, 0=none, -1=unlimited)
    txpower        Maximum transmit power in dBm (integer or float)
+   pm             WiFi Power Management setting (see below for allowed values)
    =============  ===========
+
+Constants
+---------
+
+.. data:: WLAN.PM_PERFORMANCE
+        WLAN.PM_POWERSAVE
+        WLAN.PM_DEFAULT
```

Comment:
I suggest removing `PM_DEFAULT`.  I don't think it's useful and adds unnecessarily to the code size.

Reply:
Done